src/ActionModal/components/ActionModal.vue

```diff
@@ -301,7 +301,7 @@ import { track } from "scripts/google-analytics"
 import { UserTransactionAdded } from "src/gql"
 import config from "src/../config"
 
-const ActionManager = () => import("../utils/ActionManager")
+const ActionManager = () => import(/* webpackChunkName: "ActionManager" */ "../utils/ActionManager")
```

Comment:
that looks dirty ^^' is this necessary?

Reply:
no just using chunk names for testing and debugging